connector_magento/wizards/magento_binding_backend_read.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+import contextlib
+import io
+import json
+import logging
+
+from odoo import _, api, fields, models, tools
+from odoo.exceptions import UserError
+
+from odoo.addons.component import core
+
+_logger = logging.getLogger(__name__)
+
+
+class MagentoBindingBackendRead(models.TransientModel):
+
+    _name = "magento.binding.backend.read"
+    _description = "Magento Generic Object Reader Wizard"
+
+    @api.model
+    @tools.ormcache_context("self._uid", "model_name", keys=("lang",))
+    def _get_translated_model_name(self, model_name):
+        # get the translated model name to build
+        # a meaningful model description
+        search_result = self.env["ir.model"].name_search(model_name, operator="=")
+        if search_result:
+            return search_result[0][1]
+        return self.env[model_name]._description
+
+    @api.model
+    def _default_magento_backend_id(self):
+        active_model = self._context.get("active_model")
+        if active_model and active_model == self._name:
+            # method called when the result is displayed
+            return self.env["magento.backend"].browse([])
+        if active_model and active_model != "magento.backend":
+            raise UserError(
+                _("The wizard must be launched from a magento " "backend model")
+            )
+        active_id = self._context.get("active_ids", [])
+        if len(active_id) > 1:
+            raise UserError(
+                _("The wizard must be launched on a single " "magento backend instance")
+            )
+        active_id = (active_id and active_id[0]) or self._context.get("active_id")
+        return self.env["magento.backend"].browse(active_id)
+
+    @api.model
+    def _get_magento_binding_model(self):
+        """
+        This method return a list of magento bindings for which a backend
+        adapter is registered
+        :return:
+        """
+        try:
+            components_registry = core._component_databases[self.env.cr.dbname]
+        except KeyError:
+            _logger.info(
+                "No component registry for database %s. "
+                "Probably because the Odoo registry has not been built "
+                "yet."
```

Comment:
Database name not passed.
```suggestion
                "yet.", self.env.cr.dbname
```

Reply:
Solved.

Reply:
Thanks!